### ISSUE TITLE
Dictionary leaves out common words

### ISSUE THREAD

**Issue by fresheneesz** [dropbox/zxcvbn#273]
The dictionary oddly leaves out common words like "cow" (tho "cows" seems to be in the dictionary). I recommend including a much broader dictionary step. I would imagine that actual password cracking techniques will go through words in order of assumed commonality. This vastly overestimates the security of passwords that contain words this dictionary set doesn't happen to contain. It would be best to contain a more realistic set of dictionary words.

**Comment by laughinghan** [comment]
Yeah, the "English Wikipedia" word list is weird. What dictionary do you think should be used instead?

**Comment by fresheneesz** [comment]
Well, a list like one of the following might be ideal:

* https://github.com/first20hours/google-10000-english
* https://www.ef.edu/english-resources/english-vocabulary/top-3000-words/
